content/rc/cloud-integrations/prometheus-integration.md

```diff
@@ -0,0 +1,169 @@
+---
+Title: Prometheus integration with Redis Cloud
+linkTitle: Prometheus integration
+description: To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud databases.
+weight: $weight
+alwaysopen: false
+categories: ["RC"]
+---
+
+To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud subscription.
+
+- [Prometheus](https://prometheus.io/) is an open-source systems monitoring and alerting toolkit that can scrape metrics from different sources.
+- [Grafana](https://grafana.com/) is an open-source, feature-rich metrics dashboard and graph editor that can process Prometheus data.
+
+For more information on how Prometheus communicates with Redis Enterprise Software clusters, see [Prometheus integration with Redis Enterprise Software]({{< relref "/rs/clusters/monitoring/prometheus-integration" >}}).
+
+{{< note >}}
+
+You must [enable VPC peering]({{< relref "/rc/security/vpc-peering" >}}) for your subscription to connect Prometheus and Grafana to your Redis Cloud databases. VPC peering is available only with Flexible or Annual subscriptions.  Prometheus and Grafana cannot connect to databases on Fixed or Free subscriptions.
+
+{{< /note >}}
+
+## Quick start
+
+To get started with custom monitoring with Prometheus on Docker:
+
+1. To connect Prometheus and Grafana to your Redis Cloud databases, they must be running on an machine that:
+    - Is running on the same cloud provider as your Redis Cloud subscription
+    - Is located in the same region as your Redis Cloud subscription
+    - Is connected to the VPC subnet that is peered with your Redis Cloud subscription
+    - Allows outbound connections to port 8070, so that Prometheus can scrape the Redis Cloud server for data
+    - Allows inbound connections to port 9090 for Prometheus and 3000 for Grafana
+
+    Create a directory on this machine called `prometheus`.
+
+1. Within that directory, create a file called `prometheus.yml`.
+
+1. Add the following contents to `prometheus.yml`. Replace `<instance_ip>` with the IP address of the machine and `<cluster_name>` with the FQDN portion of your database endpoint.
```

Comment:
```suggestion
1. Add the following contents to `prometheus.yml`. Replace `<instance_ip>` with the IP address of the machine and `<cluster_name>` with the FQDN portion of your database endpoint.
```
I'm not super familiar with the cloud UI, is FQDN supplied for them. We usually define it on first use in an article but if it's just listed as FQDN in the UI this should be fine.

Reply:
I would also love if we added the purpose of this file. Is it a config file?